client_sdk/go/fpc/packager.go

```diff
@@ -0,0 +1,204 @@
+package fpc
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/hyperledger/fabric/core/chaincode/persistence"
+	"github.com/hyperledger/fabric/core/chaincode/platforms/util"
+	"github.com/pkg/errors"
+)
+
+/*
+Notice: This file is has been modified for Fabric Private Chaincode.
+*/
```

Comment:
A bit a confusing comment: I guess you copy/pasted it from somewhere before starting modification? If so, you should say where you got if from ...

Reply:
the packaging code is based on `fabric-sdk-go/pkg/fab/ccpackager/lifecycle/packager.go `